### ISSUE TITLE
Elevators

### ISSUE THREAD

**Issue by DeltaEvo** [aresrpg/aresrpg#438]
https://www.youtube.com/watch?v=GBAFVm-JONI

Be able to create some smooth elevators on the map

**Comment by Sceat** [comment]
Seems also a lot of hacks 🤔

**Comment by DeltaEvo** [comment]
Yep like a lot of things in minecraft, for example the holograms also use entities
